ogb/utils/features.py

```diff
@@ -57,7 +57,7 @@ def atom_to_feature_vector(atom):
     """
     atom_feature = [
             safe_index(allowable_features['possible_atomic_num_list'], atom.GetAtomicNum()),
-            allowable_features['possible_chirality_list'].index(str(atom.GetChiralTag())),
+            safe_index(allowable_features['possible_chirality_list'], str(atom.GetChiralTag())),
```

Comment:
If you use `safe_index` for `possible_chirality_list`, you should probably add `misc` in https://github.com/snap-stanford/ogb/blob/master/ogb/utils/features.py#L4

Reply:
Done. I thought using CHI_OTHER for this would be fine, but misc makes sense too